dubbo-admin-backend/src/main/java/org/apache/dubbo/admin/config/ConfigCenter.java

```diff
@@ -129,11 +129,7 @@ MetaDataCollector getMetadataCollector() {
     }
 
     private URL formUrl(String config, String group) {
-        String protocol = config.split("://")[0];
-        String address = config.split("://")[1];
-        String port = address.split(":")[1];
-        String host = address.split(":")[0];
-        URL url = new URL(protocol, host, Integer.parseInt(port));
+        URL url = URL.valueOf(config);
         if (StringUtils.isNotEmpty(group)) {
```

Comment:
I think it should be `StringUtils.isNotBlank `

Reply:
We are building a `org.apache.dubbo.common.URL` object,
if we use `URL#valueOf()` when input string is null or malformed we got `throw new IllegalArgumentException("url == null");`.
Shall we leave the Inspection to `valueOf` internal here?